### PR TITLE
add hdinsight explorer and storage sdk

### PR DESCRIPTION
**Pull request by guiling**

### CODE REVIEW

File: HdInsightSDK/src/com/microsoft/azure/hdinsight/sdk/common/CommonConstant.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
 /**
  * Created by joezhang on 15-11-26.
  */
-public class CommonConstant {
+    public class CommonConstant {
```
Comment (ZhaminXu) [inline HdInsightSDK/src/com/microsoft/azure/hdinsight/sdk/common/CommonConstant.java]: remove leading space

File: HdInsightSDK/src/com/microsoft/azure/hdinsight/sdk/common/CommonRunnable.java [hunk 2]
```diff
@@ -10,7 +10,7 @@ public CommonRunnable(T parameter) {
         this.parameter = parameter;
     }
 
-    public abstract void runSpecificParameter(T parameter) throws E;
+    public abstract void runSpecificParameter(T parameter) throws  E;
```
Comment (ZhaminXu) [inline HdInsightSDK/src/com/microsoft/azure/hdinsight/sdk/common/CommonRunnable.java]: remove extra space before 'E'

File: src/com/microsoft/azure/hdinsight/common/PluginUtil.java [hunk 3]
```diff
@@ -2,14 +2,38 @@
...
 import com.intellij.openapi.vfs.VirtualFile;
 
+import javax.swing.*;
+
 /**
  * Created by guizha on 8/21/2015.
  */
 public class PluginUtil {
 
+    public static final Icon Product = IconLoader.getIcon("/icons/Product.png");
```
Comment (ZhaminXu) [inline src/com/microsoft/azure/hdinsight/common/PluginUtil.java]: Define those constants in another file like PluginConstants. Util should be used to define util methods

File: src/com/microsoft/azure/hdinsight/common/Resources.java [hunk 4]
```diff
@@ -1,13 +0,0 @@
-package com.microsoft.azure.hdinsight.common;
-
-import com.intellij.openapi.util.IconLoader;
-
-import javax.swing.*;
-
-/**
- * Created by zhax on 8/20/2015.
- */
-public class Resources {
```
Comment (ZhaminXu) [inline src/com/microsoft/azure/hdinsight/common/Resources.java]: this file should be kept

File: HdInsightSDK/src/com/microsoft/azure/hdinsight/sdk/common/CommonConstant.java (hunk 1 above)
Comment (guiling) [inline HdInsightSDK/src/com/microsoft/azure/hdinsight/sdk/common/CommonConstant.java]: OK

File: HdInsightSDK/src/com/microsoft/azure/hdinsight/sdk/common/CommonRunnable.java (hunk 2 above)
Comment (guiling) [inline HdInsightSDK/src/com/microsoft/azure/hdinsight/sdk/common/CommonRunnable.java]: OK

File: src/com/microsoft/azure/hdinsight/common/Resources.java (hunk 4 above)
Comment (guiling) [inline src/com/microsoft/azure/hdinsight/common/Resources.java]: OK

File: src/com/microsoft/azure/hdinsight/common/PluginUtil.java (hunk 3 above)
Comment (guiling) [inline src/com/microsoft/azure/hdinsight/common/PluginUtil.java]: OK

**Pull request merged**